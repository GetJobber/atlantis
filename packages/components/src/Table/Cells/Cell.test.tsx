import React from "react";
import renderer from "react-test-renderer";
import { Cell } from "../";

it("renders a simple cell", () => {
  const tree = renderer.create(<Cell>Ship</Cell>).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a left aligned cell", () => {
  const tree = renderer.create(<Cell align="left">Ship</Cell>).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a right aligned cell", () => {
  const tree = renderer.create(<Cell align="right">Ship</Cell>).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a center aligned cell", () => {
  const tree = renderer.create(<Cell align="center">Ship</Cell>).toJSON();
  expect(tree).toMatchSnapshot();
});
