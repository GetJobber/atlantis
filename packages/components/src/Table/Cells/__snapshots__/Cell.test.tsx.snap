// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a center aligned cell 1`] = `
<td
  className="cell center"
>
  Ship
</td>
`;

exports[`renders a left aligned cell 1`] = `
<td
  className="cell left"
>
  Ship
</td>
`;

exports[`renders a right aligned cell 1`] = `
<td
  className="cell right"
>
  Ship
</td>
`;

exports[`renders a simple cell 1`] = `
<td
  className="cell left"
>
  Ship
</td>
`;
